### DISCUSSION TITLE
Can I just integrate all the new created issue in bugzilla into jira?

### DISCUSSION THREAD

**Discussion by xgao95** [mozilla/jira-bugzilla-integration#790] [Q&A]
If could, would you pls share how to integrate all new issues and regardless the whiteboard tag? Thanks~

**Comment by leplatrem** [comment]
The current design does not allow it, but you could modify this part of the code:

https://github.com/mozilla/jira-bugzilla-integration/blob/b85d3a3c931e82b9b9489a1482d400e9ca873be2/jbi/runner.py#L55-L70

to always return the first configured action
